geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala

```diff
@@ -120,11 +120,11 @@ class ExportCommand(parent: JCommander) extends CommandWithCatalog(parent) with
 object ExportCommand {
   @Parameters(commandDescription = "Export a GeoMesa feature")
   class ExportParameters extends OptionalCqlFilterParameters {
-    @Parameter(names = Array("-F", "--format"), description = "Format to export (csv|tsv|gml|json|shp|bin)")
+    @Parameter(names = Array("-F", "--format"), description = "Format to export (csv|tsv|gml|json|shp|bin|avro)")
     var format: String = "csv"
 
-    @Parameter(names = Array("-m", "--max-features"), description = "Maximum number of features to return. default: Long.MaxValue")
-    var maxFeatures: Integer = Int.MaxValue
+    @Parameter(names = Array("-m", "--max-features"), description = "Maximum number of features to return. default: Unlimited")
```

Comment:
I think that if max features isn't set, we'll just keep returning values forever... if it is set, then it has to fit inside an Int. I could be wrong about that...